SetupDataPkg/Tools/ConfigEditor.py

```diff
@@ -863,6 +959,11 @@ def load_cfg_file(self, path, file_id, clear_config):
         for menu in self.menu_string:
             self.file_menu.entryconfig(menu, state="normal")
 
+        for menu in self.variable_menu_string:
+            self.variable_menu.entryconfig(menu, state="normal")

```

Comment:
```suggestion
        if self.admin_mode:
            for menu in self.variable_menu_string:
                self.variable_menu.entryconfig(menu, state="normal")
```



In the case that the editor was not started in admin mode, then it should not attempt to make the variable menu entries visible.

Reply:
addressed in 
https://github.com/microsoft/mu_feature_config/pull/413/commits/e6f697b5cb45f996a49bfe3faaf16de4b177f7f0